src/components/gltf-model-plus.js

```diff
@@ -232,15 +232,20 @@ function runMigration(version, json) {
   if (version < 2) {
     //old heightfields will be on the same node as the nav-mesh, delete those
     const oldHeightfieldNode = json.nodes.find(node => {
-      return (
-        node.extensions &&
-        node.extensions.MOZ_hubs_components &&
-        node.extensions.MOZ_hubs_components.heightfield &&
-        node.extensions.MOZ_hubs_components["nav-mesh"]
-      );
+      let components = null;
+      if (node.extensions && node.extensions.MOZ_hubs_components) {
+        components = node.extensions.MOZ_hubs_components;
+      } else if (node.extensions && node.extensions.HUBS_components) {
```

Comment:
HUBS_components won't have the version set. You'll want to run that separately.

Reply:
version defaults to `0` below in the case a version number isn't found.